crates/nu-command/tests/commands/return_.rs

```diff
@@ -23,3 +23,16 @@ fn early_return_if_false() {
 
     assert_eq!(actual.out, r#"100"#);
 }
+
+#[cfg(not(windows))]
+#[test]
+fn return_works_in_script_without_def_main() {
+    let actual = nu!(
+        cwd: "tests/fixtures/formats", pipeline(
+        r#"
+            exec (ls ../../../target/*/nu | get name.0) early_return.nu
```

Comment:
This seems too hacky. Does `nu early_return.nu` work?

Reply:
No, because that requires nu to be installed to the path. Without it, the test will return an error like "did you mean 'du'"?

Reply:
That's weird because it works on my end and I've used `nu` in tests before. Try `pipeline("nu early return.nu")`, it should work.

Looking up the `target` directory manually is not sound because `CARGO_TARGET_DIR` can move it to another location.

Reply:
whoops, you were right @kubouch . Thanks for the help!